pulsar-common/src/main/java/org/apache/pulsar/common/protocol/Commands.java

```diff
@@ -578,7 +578,7 @@ public static ByteBuf newSubscribe(String topic, String subscription, long consu
                 keyValue.setValue(value);
                 keyValues.add(keyValue);
             });
-            subscribe.addAllSubscriptionProperties(keyValues);
+            // subscribe.addAllSubscriptionProperties(keyValues);
```

Comment:
Same question.

Reply:
it was unintentional! glad you found it before this get merged

reverted now